modules/vaos/config/routes.rb

```diff
@@ -3,5 +3,6 @@
 VAOS::Engine.routes.draw do
   resources :appointments, only: %i[index], defaults: { format: :json }
   resources :systems, only: :index, defaults: { format: :json }
+  resources :facilities, only: :index, defaults: { format: :json }
   get 'api', to: 'apidocs#index', defaults: { format: :json }
 end
```

Comment:
Nitpick:  instead of listing the same `defaults` on every line, I think this will work:
```
VAOS::Engine.routes.draw do
  defaults format: :json do
    resources :appointments, only: %i[index]
    resources :systems, only: :index
    resources :facilities, only: :index
    get 'api', to: 'apidocs#index'
  end
end

```